react/features/base/participants/reducer.ts

```diff
@@ -208,8 +212,9 @@ ReducerRegistry.register<IParticipantsState>('features/base/participants',
             newParticipant = state.local = _participant(state.local, action);
         }
 
-        if (newParticipant) {
+        const oldParticipant = local ? state.local : state.remote.get(id);
```

Comment:
Not sure this is going to work, if the participant was already existing oldParticipant and newParticipant are going to point to the same object because they get assigned to the state above.

Reply:
Good catch

Reply:
Should be fixed now!